pandas/tests/io/json/test_readlines.py

```diff
@@ -219,3 +221,20 @@ def test_readjson_nrows_requires_lines():
     msg = "nrows can only be passed if lines=True"
     with pytest.raises(ValueError, match=msg):
         pd.read_json(jsonl, lines=False, nrows=2)
+
+
+def test_readjson_lines_chunks_fileurl(datapath):
+    # GH 27135
+    # Test reading line-format JSON from file url
+    df_list_expected = [
+        pd.DataFrame([[1, 2]], columns=["a", "b"], index=[0]),
+        pd.DataFrame([[3, 4]], columns=["a", "b"], index=[1]),
+        pd.DataFrame([[5, 6]], columns=["a", "b"], index=[2]),
+    ]
+    os_path = datapath("io", "json", "data", "line_delimited.json")
+    file_url = "file://localhost" + os_path
+    if system() == "Windows":
```

Comment:
Does generically using `Path` not work? Its not apparent why this is required in the test

Reply:
Path works. I've fixed it.